deploy-parallel.yaml

```diff
@@ -0,0 +1,29 @@
+specVersion: 1.0.0
+name: para-events
+version: 1.0.0
+description: all the parallel events subquery
+repository: 'git@github.com:parallel-finance/parallel-events-suqbl.git'
+runner:
+  node:
+    name: '@subql/node'
+    version: '0.33.0'
+  query:
+    name: '@subql/query'
+    version: '^0.12.0'
+schema:
+  file: ./schema.graphql
+network:
+  endpoint: 'wss://rpc.parallel.fi'
+  genesisHash: '0xe61a41c53f5dcd0beb09df93b34402aada44cb05117b71059cce40a2723a4e97'
+  chaintypes:
+    file: ./dist/chaintypes.js
+  dictionary: 'https://api.subquery.network/sq/subquery/parallel-heiko-dictionary'
+dataSources:
+  - kind: substrate/Runtime
+    startBlock: 900000
```

Comment:
Please confirm this @yz89

Reply:
Heiko start block: 1079000
Parallel start block: 820000